runtime/browser/demo/explorer.html

```diff
@@ -0,0 +1,28 @@
+<link rel="import" href="../../../strategy-explorer/strategy-explorer.html"></link>
+
+<style>
+  strategy-explorer {
+    display: none;
+    background-color: white;
+    width: 100%;
+  }
+</style>
+
+<script>
+  (function() {
+    const hotKey = 'e'; // + CTRL
+    let explorer = document.body.appendChild(document.createElement('strategy-explorer'));
+    let explorerVisible = false;
+    window.addEventListener('keydown', e => {
+      if (e.ctrlKey && (e.key == hotKey)) {
+        e.stopPropagation();
+        e.preventDefault();
+        explorerVisible = !explorerVisible;
+        explorer.style.display = explorerVisible ? 'block' : '';
```

Comment:
Changing the display of `demo-flow` from `flex` to `block` was creating a problem. In this formulation, strategy-explorer only manages it's own node and we don't monkey with `demo-flow` at all.